currency/manager.go

```diff
@@ -458,6 +458,33 @@ func (p *PairsManager) getPairStoreRequiresLock(a asset.Item) (*PairStore, error
 	return pairStore, nil
 }
 
+// SetDelimitersFromConfig ensures that the pairs adhere to the configured delimiters
+// Pairs.Unmarshal doesn't know what the delimiter is, so uses the first punctuation rune
+func (p *PairsManager) SetDelimitersFromConfig() error {
+	for a, s := range p.Pairs {
+		cf := s.ConfigFormat
+		if cf == nil {
+			cf = p.ConfigFormat
+		}
+		if cf == nil {
+			continue
```

Comment:
All exchanges have configs set either globally or at the asset level, `continue` and `return nil` seems a strange response to a missing field. It seems like a testing convenience when you are asking it to set something and it doesn't

Reply:
Would you suggest we return error when it's nil, then?

Reply:
Due to `CheckPairConsistency` we should never get here.
```
Unable to initialise bot engine. Error: failed to load config. Err: fatal error checking config values. Error: no exchanges enabled
Exchange BTSE: CheckPairConsistency error: exchange BTSE pair config format for asset type spot is nil
```
So we can safely make this an error, and if anyone removes/breaks `CheckPairConsistency` then it'll help.

Reply:
Sounds good! Thanks for following that up 🕵️

Reply:
Fixed 8f7559457